docs/articles/Core/DependencyInjection.md

```diff
@@ -253,7 +253,7 @@ namespace Moryx.UserAssignment
 }
 ````
 
-Know we can use the [ISubInitializer](xref:Moryx.Container.ISubInitializer) for the [DependencyRegistrationAttribute](xref:Moryx.Container.DependencyRegistrationAttribute)
+Know we can use the [ISubInitializer](xref:Moryx.Container.ISubInitializer) for the [DependencyRegistrationAttribute](../../../src/Moryx/Container/Attributes/DependencyRegistrationAttribute.cs)
```

Comment:
`ISubInitializer` does not exist anymore

Reply:
We will delete DependencyInjection.md since less information is probably better then wrong information